core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -246,24 +329,36 @@ private boolean isIntermediateTopic(final String topic) {
         return options.valuesOf(intermediateTopicsOption).contains(topic);
     }
 
-    private void deleteInternalTopics(final ZkUtils zkUtils) {
-        System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+    private void maybedeleteInternalTopics(final ZkUtils zkUtils) {
+        Boolean dryRun = options.has(dryRunOption);
+
+        if (!dryRun) {
+            System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+        } else {
+            System.out.println("Following internal/auto-created topics will be Deleted for application " + options.valueOf(applicationIdOption));
```

Comment:
I think this message is not much different from the non-dry-run one. Maybe we can have one "unified" message that can be used for both cases (to reduce the number of `if-else` branches?